src/js/controller/controller.js

```diff
@@ -408,7 +408,9 @@ Object.assign(Controller.prototype, {
             }
 
             const playReason = _getReason(meta);
-            const startTime = meta ? meta.startTime : null;
+
+            const startTime = meta.startTime = isValidNumber(meta.startTime) ? meta.startTime : _model.get('playlistItem').starttime;
```

Comment:
This changes the assumption about whether `meta` is defined or not. If it's possible for meta to be not an object, there needs to be a check before trying to assign a value to `meta.startTime`.